ts/README.md

```diff
@@ -28,10 +28,23 @@ On responses:
 
 Examples:
 
+The `flexible-event` utility accepts comma-separated lists of the
+number of windows and summary buckets per trigger data. This is
+essentially an unzipped list of two-tuples, one per trigger data.
+
 ```sh
 npm run flexible-event -- -w 1,2,3 -b 4,5,6
 ```
 
+The utility also will parse (in a rudimentary way) the source registration
```

Comment:
```suggestion
The utility also will parse the source registration
```
It's not rudimentary anymore. I made it so it's fully parsed like the actual header validator.

Reply:
oh, right, missed that.